common/src/main/java/tachyon/worker/DataServerMessage.java

```diff
@@ -293,6 +299,11 @@ public int recv(SocketChannel socketChannel) throws IOException {
       numRead = socketChannel.read(mHeader);
       if (mHeader.remaining() == 0) {
         mHeader.flip();
+
+        // These two fields are hard coded for now, until the client is converted to use netty.
+        long frameLength = mHeader.getLong(); // frame length
+        int msgRPCType = mHeader.getInt(); // RPC message type
+
         short msgType = mHeader.getShort();
```

Comment:
Maybe add a comment if this field is going to be replaced with the new `msgRPCType`?